MDANSE_GUI/Src/MDANSE_GUI/Tabs/Models/PlottingContext.py

```diff
@@ -213,15 +236,21 @@ def planes_vs_axis(self, axis_number: int) -> List[np.ndarray]:
                     fixed_argument = perpendicular_axis[plane_number]
                     slice_def[axis_number] = plane_number
                     data = self._data[tuple(slice_def)]
-                    self._planes[plane_number] = data
+                    if self._use_scaling:
+                        self._planes[plane_number] = data * self._scaling_factor
```

Comment:
Wonder if some kind of `with_scaling` function might be nicer and eliminate a lot of this `if self._use_scaling`. 

Could also make a `self.scaling_factor` `@property` which returns `1` if `not self._use_scaling`?

Alternatively, could consider tertiary assign
```python
self._planes[plane_number] = data * self._scaling_factor if self._use_scaling else data
```
or
```python
self._planes[plane_number] = data * (self._scaling_factor if self._use_scaling else 1)
```

Reply:
I've refactored the codes to clean this up a bit.